site/docs/aws.md

```diff
@@ -444,6 +444,9 @@ spark-sql --packages org.apache.iceberg:iceberg-spark3-runtime:{{ versions.icebe
 [Hive](https://docs.aws.amazon.com/emr/latest/ReleaseGuide/emr-hive.html), [Flink](https://docs.aws.amazon.com/emr/latest/ReleaseGuide/emr-flink.html),
 [Trino](https://docs.aws.amazon.com/emr/latest/ReleaseGuide/emr-presto.html) that can run Iceberg.
 
+Recently, Amazon EMR 6.5.0 [announced](https://aws.amazon.com/about-aws/whats-new/2022/01/amazon-emr-supports-apache-iceberg/) support of Apache Iceberg's Spark 3 Runtime.
+With the current release, you can use Apache Spark 3.1.2 on EMR clusters with the Iceberg table format. Please refer the [official documentation](https://docs.aws.amazon.com/emr/latest/ReleaseGuide/emr-iceberg-create-cluster.html) to create a cluster with Iceberg installed.
```

Comment:
So I understand this better myself, “the current release” is referring to EMR 6.5.0 (or EMR in general)?

I would try to leave the version comparability information in the EMR docs personally. EMR has a large number of releases that each support a unique set of software. IIRC, There are really good docs for seeing what is supported where.

So I would be in favor of deferring to the EMR docs for the exact combination of dependencies.

Maybe `Starting with EMR version 6.5.0, EMR clusters can be configured to have the necessary Apache Iceberg dependencies installed without requiring bootstrap actions. Please refer to the official documentation on how to create a cluster with Iceberg installed.`

Additionally, links to some sort of version support / dependency matrix would go a long way. Given that this information is already well documented for EMR for many things, it seems best to keep that there if possible as the support matrix is quite large and changes over time.

Reply:
Yes, the current release refers to EMR 6.5.0.

Reply:
Thanks, for the inputs! I have pushed the changes.

I don't see a version support / dependency matrix available in the EMR docs at this moment.

Reply:
That’s ok. I know support is new, and im not sure how the EMR docs get maintained / updated, but it would be great if that information could be included with new releases like the choice of Spark / Flink versions etc is included with different EMR releases (and I believe there’s a dependency matrix that goes back some number of versions).

In the mid to long term, I think that’s by far the best solution. But this makes sense for now.

Reply:
Yes, absolutely! After some searching, I could find this dependency version: https://docs.aws.amazon.com/emr/latest/ReleaseGuide/emr-650-release.html

Reply:
The link appears to be for a specific release though.